examples/2016-10-31/iot_backend/template.yaml

```diff
@@ -19,11 +19,6 @@ Resources:
         IoT:
           Type: IoTRule
           Properties:
-            AwsIotSqlVersion: 2016-03-23
```

Comment:
To fix the specific error you call out in the issue, you don't have to remove this property, you can use quotes to make it explicit that it's a string:

```yaml
            AwsIotSqlVersion: '2016-03-23'
```

Reply:
Making changes based on feedback and will submit